### DISCUSSION TITLE
Merged Branch By Mistake Module "Communicate using Markdown Problem"

### DISCUSSION THREAD

**Discussion by webkmsyed** [community/community#109853] [GitHub Education]
### Hi everyone,

Hi, everyone!
By mistake, I submit merged request without completing my pull request due to misunderstanding, but in GitHub Education I don't get updated task or Open Pull request to do same task, only its show your task rejected. Please open pull request so I practice and complete my Task.

![image](https://github.com/community/community/assets/111244923/7fd29ad9-d57b-4075-bd36-5e51d988b93f)

**Comment by Flavio13Lins** [comment]
Hi @officialkmsyed  .
Seems that it's a common issue, but it does not stop your learning process in GitHub Classromm.
Access https://education.github.com/pack/offers to see more tools of your interest and keep learning! 😊
